src/writer/clusterWorker.cpp

```diff
@@ -21,7 +21,8 @@
 
 #include "cluster.h"
 
-std::atomic<unsigned long> zim::writer::ClusterTask::waiting_task(0);
+template<>
+std::atomic<unsigned long> zim::writer::TrackableTask<zim::writer::ClusterTask>::waitingTaskCount(0);
```

Comment:
Why don't you define this and its `IndexTask` sibling in `writer/workers.h`?

```c++
template<class T>
std::atomic<unsigned long> zim::writer::TrackableTask<T>::waitingTaskCount(0);
```

Reply:
It was to avoid a multiple definition of the static member.
But it seems that with template, it is a bit different and the actual definition is made only once.

Fixed with a fixup! commit